Code/Tools/AssetProcessor/native/tests/assetmanager/AssetManagerTestingBase.h

```diff
@@ -43,9 +43,9 @@ namespace UnitTests
         }
 
         void WaitForFinish()
-        {
-            m_signal.acquire();
-        }
+            {
```

Comment:
why did this get tabbed in?

Reply:
Probably something with VS auto formatting, mine's been acting weird lately.  Fixed